src/components/menu/__tests__/editor-mode-toolbar.test.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/no-named-as-default */
+import React from 'react'
+import { shallow } from 'enzyme'
+import { EditorModeToolbarUnconnected } from '../editor-mode-toolbar'
+import EditorModeTitle from '../editor-mode-title'
+import ViewControls from '../view-controls'
+import IodideLogo from '../../../shared/iodide-logo'
+
+describe('<EditorModeToolbarUnconnected', () => {
+  let wrapper
+  beforeEach(() => {
+    wrapper = shallow(<EditorModeToolbarUnconnected />)
+  })
+  it('Should render EditiorModeTitle', () => {
+    const viewMode = ['EXPLORE_VIEW', '']
+    viewMode.forEach((value) => {
+      wrapper.setProps({ viewMode: value })
+      expect(wrapper.find(EditorModeTitle)).toHaveLength(1)
+      expect(wrapper.find(ViewControls)).toHaveLength(1)
+      expect(wrapper.find(IodideLogo)).toHaveLength(1)
+    })
+  })
+  it('should have display none property', () => {
```

Comment:
in these descriptions, say more about the condition you're testing, e.g.:
'should have display none when viewMode !== EXPLORE_VIEW'